projects/Mallard/src/hooks/use-edition-provider.tsx

```diff
@@ -221,21 +221,11 @@ export const defaultEditionDecider = async (
 	} else {
 		// Get the correct edition for the device locale
 		const autoDetectedEdition = localeToEdition(locale, editionsList);
-
-		if (autoDetectedEdition) {
-			await setEdition(
-				autoDetectedEdition,
-				setDefaultEdition,
-				setSelectedEdition,
-			);
-		} else {
-			// auto detected edition was not possible, set default edition
-			await setEdition(
-				BASE_EDITION,
-				setDefaultEdition,
-				setSelectedEdition,
-			);
-		}
+		await setEdition(
+			autoDetectedEdition,
+			setDefaultEdition,
+			setSelectedEdition,
+		);
```

Comment:
setEditions defaults to base_editon to the else seems irrelevant but would appreciate a second pair of eyes...

Reply:
I think we might need it in still unless I'm missing something  - can you expand on why it might be irrelevant?